lib/cinch/plugins/wordgames/game.rb

```diff
@@ -35,14 +32,25 @@ def number_of_guesses_phrase
     end
 
   protected
-    def guess_correct?(word, response)
-      if @word == word
-        response.game_won
-        true
+    def blank?(bound)
+      bound == Blank_str
+    end
+
+    def evaluate_guess!(word)
```

Comment:
I'm confused why you used '!' suffix on `evaluate_guess!` and `guess!` 

! indicate that the method will modify the object it's called on, but that does not seem to be the case here. I still feel like a Ruby n00b, what do you think?